BasicTest/Program.cs

```diff
@@ -35,12 +35,12 @@ static void MassiveBulkOpsAsync(int AsyncOpsQty, bool preserveOrder, bool withCo
                 var conn = muxer.GetDatabase();
                 muxer.Wait(conn.PingAsync());
 
-#if DNXCORE50
+#if CORE_CLR
                 int number = 0;
 #endif
                 Action<Task> nonTrivial = delegate
                 {
-#if !DNXCORE50
+#if !CORE_CLR
```

Comment:
@NickCraver I see -- yeah, don't worry about it. If what you have works no point in messing around unnecessarily.